pr.codecept.js

```diff
@@ -1,27 +1,57 @@
+const { setHeadlessWhen } = require('@codeceptjs/configure');
+
 const { pageObjects, getChunks } = require('./codeceptConfigHelper');
 
+require('dotenv').config();
+
+// by default run in headless mode
+setHeadlessWhen(!(process.env.SHOW_BROWSER === 'true'));
+
 exports.config = {
   output: 'tests/output',
   helpers: {
     Playwright: {
+      show: true,
       url: process.env.PMM_UI_URL || 'http://127.0.0.1/',
-      restart: true,
+      restart: false,
```

Comment:
it will restart browser between tests, I think it is a  sign of flaky tests. I think it is better to fix test and avoid this workaround